### ISSUE TITLE
 Pop!_OS not detected as Debian-like

### ISSUE THREAD

**Issue by abhimanyu003** [KasperskyLab/TinyCheck#9]
When running it on Pop!_OS it's not detected as  Debian-like system.

The output for ID_LIKE is `ID_LIKE="ubuntu debian"`

```sh
❯ sudo cat /etc/os-release
NAME="Pop!_OS"
VERSION="20.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.04 LTS"
VERSION_ID="20.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
LOGO=distributor-logo-pop-os
```

**Comment by felixaime** [comment]
Hello abhimanyu003,

In that case, patch your /etc/os-release by just ID_LIKE="debian".
I don't know how many forks there is of ubuntu. If many users encounter that issue I'm gonna to patch the install.sh

Have a good day!
Félix.
